sql/clients_last_seen_v1.sql

```diff
@@ -0,0 +1,30 @@
+WITH current_sample AS (
+  SELECT
+    submission_date_s3 AS last_seen_date,
+    * EXCEPT (submission_date_s3)
+  FROM
+    telemetry.clients_daily_v6
+  WHERE
+    submission_date_s3 = @submission_date
+), previous AS (
+  SELECT
+    * EXCEPT (submission_date,
+      generated_time)
+  FROM
+    analysis.last_seen_v1
```

Comment:
Is the dataset prefix here intended?

Reply:
whoops no

Reply:
i take that back, this one is needed because it doesn't match the on line 6 above. i will figure out how to make this better as i test it.